internal/action/aliases.go

```diff
@@ -27,59 +25,3 @@ func (s *Action) AliasesPrint(c *cli.Context) error {
 
 	return nil
 }
-
-// AliasesAdd adds a single alias to a domain.
-func (s *Action) AliasesAdd(c *cli.Context) error {
-	ctx := ctxutil.WithGlobalFlags(c)
-	domain := c.Args().First()
-	alias := c.Args().Get(1)
-
-	if domain == "" || alias == "" {
-		return exit.Error(exit.Usage, nil, "Usage: %s alias add <domain> <alias>", s.Name)
-	}
-
-	if err := pwrules.AddCustomAlias(domain, alias); err != nil {
-		return err
-	}
-
-	out.Printf(ctx, "Added alias %q to domain %q", alias, domain)
-
-	return nil
-}
-
-// AliasesRemove removes a single alias from a domain.
-func (s *Action) AliasesRemove(c *cli.Context) error {
-	ctx := ctxutil.WithGlobalFlags(c)
-	domain := c.Args().First()
-	alias := c.Args().Get(1)
-
-	if domain == "" || alias == "" {
-		return exit.Error(exit.Usage, nil, "Usage: %s alias remove <domain> <alias>", s.Name)
-	}
-
-	if err := pwrules.RemoveCustomAlias(domain, alias); err != nil {
-		return err
-	}
-
-	out.Printf(ctx, "Remove alias %q from domain %q", alias, domain)
-
-	return nil
-}
-
-// AliasesDelete remove an alias mapping for a domain.
-func (s *Action) AliasesDelete(c *cli.Context) error {
-	ctx := ctxutil.WithGlobalFlags(c)
-	domain := c.Args().First()
-
-	if domain == "" {
-		return exit.Error(exit.Usage, nil, "Usage: %s alias delete <domain>", s.Name)
-	}
-
-	if err := pwrules.DeleteCustomAlias(domain); err != nil {
-		return err
-	}
-
-	out.Printf(ctx, "Remove aliases for domain %q", domain)
-
-	return nil
-}
```

Comment:
Because I'm moving domain aliases to the config.
Why in this PR? It's kind of a feasibility test. Sure it might have deserved it's own PR but a this point I'm too lazy to tear it apart again.